apps/api/src/index.ts

```diff
@@ -79,21 +82,9 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
 // Define different types of protected routes with their authentication needs
-const agentApiKeyRoutes = [
-  "/api/agent",
-  "/api/trade", // Trade operations use agent API keys
-];
-
-const userSessionRoutes = [
-  "/api/user", // User operations use SIWE sessions
-];
-
-const legacyRoutes = [
```

Comment:
note: i removed the `legacyRoutes` and duplicative auth middleware. this was left over from the refactor. i just wanna double check the auth settings are correct here.